auth-jwt-service/pom.xml

```diff
@@ -0,0 +1,124 @@
+    <artifactId>stargate</artifactId>
+    <version>0.0.23-SNAPSHOT</version>
+  </parent>
+  <groupId>io.stargate.auth.jwt</groupId>
+  <artifactId>auth-jwt-service</artifactId>
+  <dependencies>
+    <dependency>
+      <groupId>io.stargate.auth</groupId>
+      <artifactId>authnz</artifactId>
+      <version>${project.version}</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.stargate.db</groupId>
+      <artifactId>persistence-api</artifactId>
+      <version>${project.version}</version>
+      <scope>provided</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>com.google.guava</groupId>
+          <artifactId>guava</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>org.json</groupId>
+      <artifactId>json</artifactId>
+      <version>20190722</version>
+    </dependency>
+
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>log4j-over-slf4j</artifactId>
+      <scope>runtime</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.nimbusds</groupId>
+      <artifactId>nimbus-jose-jwt</artifactId>
+      <version>9.0.1</version>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <configuration>
+          <unpackBundle>true</unpackBundle>
+          <instructions>
+            <Bundle-Name>AuthJWTService</Bundle-Name>
+            <Bundle-Description>AuthJWTService component for project</Bundle-Description>
+            <Bundle-SymbolicName>io.stargate.auth.jwt</Bundle-SymbolicName>
+            <Bundle-Activator>io.stargate.auth.jwt.AuthJWTServiceActivator</Bundle-Activator>
+            <Import-Package><![CDATA[
+                        io.stargate.auth,
+                        org.osgi.framework,
+                        io.stargate.core,
+                        io.stargate.db,
+                        com.codahale.metrics
+                        ]]></Import-Package>
+            <Export-Package>io.stargate.auth.jwt</Export-Package>
```

Comment:
Do we  expect these classes to be referenced directly? I thought all access would be through `io.stargate.auth.*` so this export seems unnecessary to me. WDYT?

Reply:
Right. That's a good point. Probably need the same for the table-based stuff too.